mantidproject/mantid
# Tomography GUI improvements in "ROI" tab for color maps

Adds a couple of widgets in the "ROI etc" tab of the tomography GUI: Browse individual images, and color bar widget.

**To test:**
- Open "Custom Interfaces"=>"Diffraction"=>"Tomographic Reconstruction"
- Load an individual image with the "Image..." button that you find at the top. For example: `FITS_small_01.fits` from the unit test data.
- The unit test of the ROI widget has been updated and it should pass.
- Check that:
  - You can load maps by double-clicking on the colorbar (or the scale and numbers next to the bar)
  - You can change between linear/log/power scales, change the min/max, etc.
- This modifies the minimum sizes of the color bar widget, which is also used in the slice viewer. Open the slice viewer and check that the color bar still displays correctly there. For example open it for this file from the unit-test data: `SEQ_NDHW.nxs`.

The meaning of the two tick boxes at the bottom of the color bar can be a bit misleading. It might need some clarification/changes, but as this is also used in the slice viewer, I'm not changing it here.
- Autoscale to current slice: if it is on, the interface will get the min/max values from the image, on loading a new image or a new color map. But you can still change them manually in the spin boxes
- Autoscale on load: if t is on, you won't be able to edit the min/max manually. This option probably has little applicability in this context.

Fixes #16048.

Release note: https://github.com/mantidproject/mantid/commit/6e2d4ad6222e6907827d708704a227cb6d1a37db

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
